README.md

```diff
@@ -189,3 +190,16 @@ Once you have the images built you can use them in the previous commands by repl
 
 For a more sophisticated deployment which includes bidirectional header sync, message passing,
 monitoring dashboards, etc. see the [Deployments README](./deployments/README.md).
+
+## Community
+
+Main hangout for the community is [Element](element.io). Element (formerly Riot) is a chat server
```

Comment:
```suggestion
Main hangout for the community is [Element](https://element.io/) (formerly Riot). Element is a chat server
```

Right now it thinks it's a subpage of the README, which it's not (I also didn't realize Markdown would default to this without the "full" URL).

Reply:
@adoerr I think you forgot to add this suggestion to "batch" before commiting. I've fixed the URL though